qiskit/visualization/latex.py

```diff
@@ -299,7 +299,8 @@ def _get_image_depth(self):
         columns = 2
 
         # add extra column if needed
-        if self.cregbundle and (self.ops[0][0].name == "measure" or self.ops[0][0].condition):
+        if self.cregbundle and (self.ops and self.ops[0] and
```

Comment:
Three questions here: 
- Can any list of ops (like self.ops[0]) be empty? Should we keep the code defensive and and check it, or can we relax it and just check that self.ops isn't empty?
- Can the list of lists, or any of the lists ever be `None`?
- If it's not possible*, or for the sake of readability, would you prefer an explicit check using `len(self.ops)` etc...?

(*) in that case, I guess it's better to fail spectacularly if `self.ops` or `self.ops[0]` are not lists...